### ISSUE TITLE
FBA Blackhurst_IO 2002 in FLOWSA no longer runs with the changes to make_url_request()

### ISSUE THREAD

**Issue by catherinebirney** [USEPA/esupy#37]
@bl-young I have not looked into the cause of the error. But when running Blackhurst_IO_2002, I get the error "requests.exceptions.HTTPError: 503 Server Error: Service Temporarily Unavailable for url" - there is no issue with the URL, I can manually download the pdf without issue using the same link that is in FLOWSA

**Comment by bl-young** [comment]
Yes I noticed that it was having issues despite the url working. Are you sure it was working before this change?

**Comment by catherinebirney** [comment]
The last time I can confirm Blackhurst_IO worked was 2/16/22 for v1.1 of FLOWSA, I'm not sure what change caused the FBA to stop working
